create_aio_app/template/{{cookiecutter.project_name}}/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
     ports:
       - 8080:8080
       - 8081:8081
-    command: adev runserver ./{{ cookiecutter.project_name }}/__main__.py -p 8080
+    command: make {%- if cookiecutter.use_postgres == 'y' %} wait_postgres -- {% endif %} adev
```

Comment:
will be better if you will be use `if` construction  in `Makefile`.

Reply:
It will be even better if for simple things nobody used 3 unnecessary abstract boxes... One tool is enough.